canonical/maas-ui
# WIP feat(MachineDetails): New action button group on Machine Details page MAASENG-1223

## Done

- Created new action button group for machine details page
- Removed "Take Action" button
- Removed power options button

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Click on a machine and go to its details page
- Ensure that clicking on each action opens the correct action form
- Ensure that actions can be submitted (error returns are to be expected for some actions)

## Fixes

Fixes https://warthogs.atlassian.net/browse/MAASENG-1223

## Screenshots

### Old
![image](https://user-images.githubusercontent.com/35104482/211866355-2803afeb-995f-41a2-9e7e-633caea65904.png)

### New
![image](https://user-images.githubusercontent.com/35104482/211866470-5540870d-558e-42fa-9b9d-9254bed3bced.png)
